src/welle-gui/i18n/es_ES.ts

```diff
@@ -0,0 +1,1766 @@
+    </message>
+    <message>
+        <location filename="../QML/settingpages/RTLTCPSettings.qml" line="84"/>
+        <source>Enter host name</source>
+        <translation>Introducir nombre de host</translation>
+    </message>
+    <message>
+        <location filename="../QML/settingpages/RTLTCPSettings.qml" line="88"/>
+        <source>Host name</source>
+        <translation>Nombre de host</translation>
+    </message>
+</context>
+<context>
+    <name>RadioView</name>
+    <message>
+        <location filename="../QML/RadioView.qml" line="12"/>
+        <source>Service Overview</source>
+        <translation>Descripción general</translation>
+    </message>
+</context>
+<context>
+    <name>RawFileSettings</name>
+    <message>
+        <location filename="../QML/settingpages/RawFileSettings.qml" line="12"/>
+        <source>RAW file settings</source>
+        <translation>Ajustes de fichero RAW</translation>
+    </message>
+    <message>
+        <location filename="../QML/settingpages/RawFileSettings.qml" line="42"/>
+        <source>Open RAW file</source>
+        <translation>Abrir fichero RAW</translation>
+    </message>
+    <message>
+        <location filename="../QML/settingpages/RawFileSettings.qml" line="46"/>
+        <source>Currently shown under Android</source>
+        <translation>Actualmente se muestra en Android</translation>
+    </message>
+    <message>
+        <location filename="../QML/settingpages/RawFileSettings.qml" line="69"/>
+        <source>Selected file:</source>
+        <translation>Fichero elegido:</translation>
+    </message>
+</context>
+<context>
+    <name>RawRecorder</name>
+    <message>
+        <location filename="../QML/expertviews/RawRecorder.qml" line="11"/>
+        <source>I/Q RAW Recorder (experimental)</source>
+        <translation>Grabadora I/Q RAW (experimental)</translation>
+    </message>
+    <message>
+        <location filename="../QML/expertviews/RawRecorder.qml" line="19"/>
+        <source>Ring buffer length [s]</source>
+        <translation>Longitud del búfer de anillo [s]</translation>
+    </message>
+    <message>
+        <location filename="../QML/expertviews/RawRecorder.qml" line="33"/>
+        <source>Save ring buffer</source>
+        <translation>Guardar búfer de anillo</translation>
+    </message>
+    <message>
+        <location filename="../QML/expertviews/RawRecorder.qml" line="33"/>
+        <source>Init</source>
+        <translation>Init</translation>
+    </message>
+    <message>
+        <location filename="../QML/expertviews/RawRecorder.qml" line="47"/>
+        <source>Ring buffer size (roughly): </source>
+        <translation>Tamaño de búfer de anillo</translation>
+    </message>
+</context>
+<context>
+    <name>ServiceDetails</name>
+    <message>
+        <location filename="../QML/expertviews/ServiceDetails.qml" line="10"/>
+        <source>Service Details</source>
+        <translation>Dtalles de la estación</translation>
```

Comment:
Typo?

Reply:
Hi Andreas,

Yes, it is a typo. It should be "Detalles de la estación". File updated.

Regards,
Nándor